Fabric.Identity.API/Models/ModelExtensions.cs

```diff
@@ -76,7 +78,7 @@ public static IS4.Client ToIs4ClientModel(this Client client)
                     Claims = client.Claims,
                     AlwaysSendClientClaims = client.AlwaysSendClientClaims,
                     AlwaysIncludeUserClaimsInIdToken = client.AlwaysIncludeUserClaimsInIdToken,
-                    PrefixClientClaims = client.PrefixClientClaims,
+                    ClientClaimsPrefix = client.PrefixClientClaims ? "client_" : null,

```

Comment:
Was the prefixclientclaims set to "client_" before? Is this just to distinguish client claims from user claims?

Reply:
When we were updating IdentityServer, there were quite a few places where they updated types on objects we were using. In this case, it used to be a boolean value, but now it is a string. Looking up in the former code, it added `client_` as the prefix if the boolean was set to true. Although the new structure is more flexible, our code was built to work with the old structure. In particular, we had a Client model class that had a boolean for this parameter. The ternary operator above is there to try and accommodate the new structure in IdentityServer without rewriting the functionality we currently have.